osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -582,24 +582,12 @@ private void load(OsuColour colours, AudioManager audioManager, INotificationOve
 
                     volumeOverlay.IsMuted.Value = false;
 
-                    double aggregateVolumeTrack = audioManager.Volume.Value * audioManager.VolumeTrack.Value;
-
                     // Check values before resetting, as the user may have only had mute enabled, in which case we might not need to adjust volumes.
                     // Note that we only restore halfway to ensure the user isn't suddenly overloaded by unexpectedly high volume.
-                    if (aggregateVolumeTrack <= volume_requirement)
-                    {
-                        // Prioritize increasing music over master volume as to avoid also increasing effects volume.
-                        const double target = 0.5;
-                        double result = target / Math.Max(0.01, audioManager.Volume.Value);
-
-                        if (result > 1)
-                        {
-                            audioManager.Volume.Value = target;
-                            audioManager.VolumeTrack.Value = 1;
-                        }
-                        else
-                            audioManager.VolumeTrack.Value = result;
-                    }
+                    if (audioManager.Volume.Value <= volume_requirement)
+                        audioManager.Volume.Value = 0.5;
+                    if (audioManager.VolumeTrack.Value <= volume_requirement)
+                        audioManager.VolumeTrack.Value = 0.5;
 
```

Comment:
Okay, but now take the 10% of 10% example. That'd be a low enough aggregate volume to trigger the notification, but neither of the volume levels meet the `volume_requirement` threshold of 0.01, and so the volume is going to be left exactly as it is.

Reply:
then just change it to use <= 0.5 instead of the const

Reply:
That sounds pretty reasonable, honestly.

Reply:
@myQwil can you check again and confirm everything is correct now?